api/datahub.oas3.yml

```diff
@@ -722,6 +860,71 @@ components:
     QueryResponse:
       type: object
 
+    LoginProviders:
+      type: array
+      items:
+        $ref: '#/components/schemas/LoginProvider'
+
+
+    LoginProvider:
+      properties:
+        name:
+          type: string
+          description: Unique identifying name for this provider
+        type:
+          type: string
+          enum:
+            - bearer
+            - basic
+          description: Must be either bearer (jwt) or basic (username+password)
+        user:
+          type: object
+          $ref: '#/components/schemas/Value'
+          description: If type is "basic" then this is the username and must be present
+        password:
+          type: object
+          $ref: '#/components/schemas/Value'
+          description: If type is "basic" then this is the password and must be present
+        key:
+          type: object
+          $ref: '#/components/schemas/Value'
+          description: If type is "bearer" then this is the id of the client
+        secret:
+          type: object
+          $ref: '#/components/schemas/Value'
+          description: If type is "bearer" then this is the secret of the client
+        audience:
+          type: object
+          $ref: '#/components/schemas/Value'
+          description: If type is "bearer" then this is the audience the token is requested for.
+        grantType:
+          type: object
+          $ref: '#/components/schemas/Value'
+          description: |
+            If type is "bearer" then this is the type of the token asked for, this is client_credential for
+            m2m tokens, app_credential for user application tokens.
+        endpoint:
```

Comment:
Should we state here that this must be an OAuth compliant client credential flow endpoint:
https://datatracker.ietf.org/doc/html/rfc6749#section-4.4